static/resources/minikube-values.yaml

```diff
@@ -29,72 +29,72 @@ global:
       secretName: ''
       # Key in the above secret with the certificate
       secretKey: db2admin.arm
-  apps:
-    config:
-      curam:
-        enabled: true
-        resources:
-          limits:
-            cpu: 2
-            memory: 4Gi
-          requests:
-            cpu: 1
-            memory: 2Gi
-      rest:
-        enabled: false
-      citizenportal:
-        enabled: false
-      curamwebservices:
-        enabled: false
-      curambirtviewer:
-        enabled: false
   ingress:
     enabled: true
     tlsSecretName: testkube-tls
     generateCert: true
     hostname: minikube.local
+  apps: {}
+    # config:
+    #   curam:
+    #     enabled: true
+    #     resources:
+    #       limits:
+    #         cpu: 2
+    #         memory: 4Gi
+    #       requests:
+    #         cpu: 1
+    #         memory: 2Gi
+    #   rest:
+    #     enabled: false
+    #   citizenportal:
+    #     enabled: false
+    #   curamwebservices:
+    #     enabled: false
+    #   curambirtviewer:
+    #     enabled: false
 
-# Override values in 'apps' Chart
-apps:
-  replicaCount: 1
-  initialDelaySeconds: 60
-  loginConfig:
-    trace: true
-  sessionConfig:
-    sessionTimeout: 30m
-    transactionTimeout: 3m
-  defaultResources:
-    requests:
-      cpu: 0.5
-      memory: 768Mi
-    limits:
-      cpu: 0.75
-      memory: 1Gi
+# # Override values in 'apps' Chart
+apps: {}
+#   replicaCount: 1
+#   initialDelaySeconds: 60
+#   loginConfig:
+#     trace: true
+#   sessionConfig:
+#     sessionTimeout: 30m
+#     transactionTimeout: 3m
+#   defaultResources:
+#     requests:
+#       cpu: 0.5
+#       memory: 768Mi
+#     limits:
+#       cpu: 0.75
+#       memory: 1Gi
 
-# Override values in 'batch' Chart
-batch:
-  programs:
-    queued:
-      schedule: "*/30 * * * *"
+# # Override values in 'batch' Chart
+batch: {}
+#   programs:
+#     queued:
+#       schedule: "*/30 * * * *"
 
-# Override values in 'uawebapp' Chart
-uawebapp:
-  replicaCount: 1
-  imageConfig:
-    library: ''
-    name: ''
-    tag: latest
-  readinessProbe:
-    path: /universal/
-  ingressPath: /universal
+# # Override values in 'uawebapp' Chart
+uawebapp: {}
+#   replicaCount: 1
+#   imageConfig:
+#     library: ''
+#     name: ''
+#     tag: latest
+#   readinessProbe:
+#     path: /universal/
+#   ingressPath: /universal
 
-# Override values in 'web' Chart
-web:
-  replicaCount: 1
-  readinessProbe:
-    path: /CuramStatic/
-  ingressPath: /CuramStatic
+# # Override values in 'web' Chart
+web: {}
+#   replicaCount: 1
+#   readinessProbe:
+#     path: /CuramStatic/
+#   ingressPath: /CuramStatic
 
-# Override values in 'xmlserver' Chart
-xmlserver:
-  replicaCount: 1
+# # Override values in 'xmlserver' Chart
+xmlserver: {}
+#   replicaCount: 1
```

Comment:
Are we okay with the comments at the start of the line here, originally they were one tabulation away from it, no?

e.g.
```
xmlserver: {}
  replicaCount: 1
  #  replicaCount: 1 
```
instead of

```
xmlserver: {}
  replicaCount: 1
#   replicaCount: 1 
```

Reply:
Comment is applying for the whole file actually

Reply:
I think this is fine